include/FNA3D.h

```diff
@@ -166,7 +166,8 @@ typedef enum FNA3D_CubeMapFace
 typedef enum FNA3D_BufferUsage
 {
 	FNA3D_BUFFERUSAGE_NONE,
-	FNA3D_BUFFERUSAGE_WRITEONLY
+	FNA3D_BUFFERUSAGE_WRITEONLY,
+	FNA3D_BUFFERUSAGE_COMPUTE_EXT /* SDL shader extension only */
```

Comment:
Does this mean 'usable by compute AND write only'? Or is it a bit

Reply:
This value is actually completely ignored on every backend, so setting COMPUTE_EXT on the SDL backend is the only place this would actually matter.